lib/mayaUsd/utils/utilFileSystem.cpp

```diff
@@ -27,35 +27,14 @@
 
 #include <ghc/filesystem.hpp>
 
-#include <random>
-#include <system_error>
-
-namespace {
-std::string generateUniqueName()
```

Comment:
@dj-mcg How this change is related to AR 2.0?

Reply:
Yep - good catch - an internal patched change snuck in with my push. I'll be pushing a new version momentarily